examples_extra/VideoTableView/Sample/NicCageNode.mm

```diff
@@ -167,8 +167,11 @@ - (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
 {
   CGSize videoNodeSize = _isImageEnlarged ? CGSizeMake(2.0 * kImageSize, 2.0 * kImageSize)
                                           : CGSizeMake(kImageSize, kImageSize);
-  _videoNode.size = ASRelativeSizeRangeMakeWithExactCGSize(videoNodeSize);
-  _textNode.flexShrink = 1.0;
+  
+  [_videoNode.style setWidth:ASDimensionMake(videoNodeSize.width)];
+  [_videoNode.style setHeight:ASDimensionMake(videoNodeSize.height)];
```

Comment:
Can you please change that to: `_videoNode.style.preferredSize = videoNodeSize;`